### ISSUE TITLE
Working Platform dropdown (WPF/Console/IOS/Android) is not getting visual keyboard focus and no indication of getting focus on it in High contrast black mode

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#383]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Navigate to working platform dropdown(WPF/Console/IOS/Android) beside "Help" menu item and observe the visual focus and selection state.

### Expected Behavior
Dropdown should get visual focus or indication of focus in High contrast mode.
### Actual Behavior
Working Platform dropdown (WPF/Console/IOS/Android) is not getting visual keyboard focus and not following selected element standards in High contrast black mode.
No visual indication of getting focus on to the dropdown.
Note: In high contrast white mode dotted rectangular focus appears on working platform dropdown refer HCW.png screenshot.
### User Impact
Low vision user will not know whether the element got focus or not.
### Screenshots
![19 hcb](https://user-images.githubusercontent.com/22956130/40763581-f9b2e962-64c2-11e8-9f38-0c9e943f735f.png)
![19 hcw](https://user-images.githubusercontent.com/22956130/40763582-f9dcedca-64c2-11e8-829d-1bf8b3b86db0.png)
![19 normal](https://user-images.githubusercontent.com/22956130/40763583-fa056c5a-64c2-11e8-8da1-f01ba0949100.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
